lib/ansible/plugins/lookup/lastpass.py

```diff
@@ -59,13 +59,13 @@ def logged_in(self):
         out, err = self._run(self._build_args("logout"), stdin="n\n", expected_rc=1)
         return err.startswith("Are you sure you would like to log out?")
 
-    def _run(self, args, stdin=None, expected_rc=0):
+    def _run(self, args, stdin='', expected_rc=0):
         p = Popen([self.cli_path] + args, stdout=PIPE, stderr=PIPE, stdin=PIPE)
-        out, err = p.communicate(stdin)
+        out, err = p.communicate(stdin.encode('ascii'))
         rc = p.wait()
         if rc != expected_rc:
             raise LPassException(err)
-        return out, err
+        return out.decode('utf-8'), err.decode('utf-8')
```

Comment:
to_text here is good practice:
```suggestion
        return to_text(out, errors='surrogate_or_strict'), to_text(err, errors='surrogate_or_strict')
```

Reply:
Fixed. Thanks.